### DISCUSSION TITLE
Mejorar portafolio

### DISCUSSION THREAD

**Discussion by pyjavo** [Scot3004/secorto_web#66] [Ideas]
Me parece que para mejorar tu porfafolio como Software Engineer o Especialista en seguridad informática, éste debería estar enfocado en ello por lo que la experiencia de voluntariado en la PyCon debería ir en otro apartado. 

Como alguien enfocado en testing, debería de haber portafolio relacionado a dicho tema.

Por útimo, el portafolio se supone que sirve para mostrar tus mejores trabajos, no para mostrar experimentos. Por lo que tampoco considero pertinente incluir el website de scot3004 como portafolio.

Espero haber ayudado en algo. 
PySaludos

**Comment by Scot3004** [comment]
Tienes toda la razón, me toca organizar varias cosas del sitio y esta es una de las más notables

**Comment by Scot3004** [comment]
@pyjavo actualice lo que comentas en este issue de momento solo están los proyectos en los que estoy trabajando, sin embargo eventualmente agregare lo aprendido de mis anteriores trabajos

**Comment by LuiJota** [comment]
También estoy buscando crear un portafolio para una web decandados y cerraduras de seguridad. Estoy comenzando por abarcar una mayor información por aquí.

**Reply by Scot3004** [reply]
genial que tengas un sitio de candados y eso, en cuanto al portafolio que tienes pensado hacer, y sobre que tecnologias tienes montado tu sitio?
